### DISCUSSION TITLE
Does getter set persist to true?

### DISCUSSION THREAD

**Discussion by loss-and-quick** [mikro-orm/mikro-orm#4357] [Q&A] [answered]
The getter and setter properties are described inside the types
But what do they actually do?
I mean, do they make serializable getter(setter)?

**Comment by B4nan** [comment, marked as answer]
Is your property a getter, setter, or both? That is where you use those. They are set automatically if you use decorators, you don't have to care about them anyhow in that case.

This is mostly used for hydration and change tracking, e.g. when there is a getter without setter, such values won't be mapped or assigned - and also ignored during hydration if they also have `persist: false` (but without that, it will be hydrated, overriding the getter with a database value). 

So to answer your first question - nope, it does not set persist to true, persist is true by default. Not sure what you meant by the other question.

Here is an example of where those are used, check the `EntitySchema` tab in that example:

https://mikro-orm.io/docs/defining-entities#virtual-properties

**Reply by loss-and-quick** [reply]
I can not setup one decorator to getter and setter,so i must use one them (i will send an example)
And why getter and setter properties doesn't existing at Property's docs

**Reply by B4nan** [reply]
> I can not setup one decorator to getter and setter,so i must use one them (i will send an example)

Just put the decorator on the getter, that's enough.
